src/pages/management/webhooks/index.mdx

```diff
@@ -9,7 +9,7 @@ apiVersion: "3.4"
 ---
 
 This document contains a reference of available webhooks. Webhooks are helpful for building applications that execute particular behavior in response to various actions, allowing you to create automated flows. 
-In orderd to receive webhooks, you need to [register](/management/configuration-api/v3.4/#register-webhook) them first.
+In order to receive webhooks, you need to [register](/management/configuration-api/v3.4/#register-webhook) them first.
```

Comment:
FYI - I took a chance and just corrected the typo, so below 3 files are connected with the typo correction.